settings.gradle.kts

```diff
@@ -1,50 +1,17 @@
-import org.gradle.api.internal.FeaturePreviews
-
-pluginManagement {
-    plugins {
-        id("org.jetbrains.kotlin.jvm") version "1.8.10"
-        id("com.github.johnrengelman.shadow") version "8.1.0"
-    }
-}
 plugins {
-    id("com.gradle.enterprise").version("3.12.3")
-    id("com.gradle.common-custom-user-data-gradle-plugin").version("1.8.2")
+    id("com.gradle.enterprise").version("3.13")
+    id("com.gradle.common-custom-user-data-gradle-plugin").version("1.10")
 }
 
-dependencyResolutionManagement {
-    repositories {
-        maven {
-            name = "Gradle public repository"
-            url = uri("https://repo.gradle.org/gradle/public")
-            content {
-                includeGroup("net.rubygrapefruit")
-                includeModule("classycle", "classycle")
-                includeModule("flot", "flot")
-                includeModule("org.gradle", "gradle-tooling-api")
-                includeModuleByRegex("org.gradle", "docs-asciidoctor-extensions(-base)?")
-            }
-        }
-        google {
-            content {
-                includeGroup("com.android.databinding")
-                includeGroupByRegex("com\\.android\\.tools(\\.[a-z.\\-]*)?")
-            }
-        }
-        maven {
-            name = "CHAMP libs"
-            url = uri("https://releases.usethesource.io/maven/")
-            mavenContent {
-                includeGroup("io.usethesource")
-            }
-        }
-        mavenCentral()
```

Comment:
Why not leave this to be confighred for the entire build from here?

Reply:
That's a good point. I'll move the `mavenCentral` declaration back to settings.